data/navLinksData.js

```diff
@@ -0,0 +1,22 @@
+export const links = [
+  {
+    key: 0,
```

Comment:
Please use ID, instead of key. Because from the backend we often get ID, so we should stick to one convention.